### PR TITLE
added countdown component

### PR DESCRIPTION
**Pull request by kstar20001**

### CODE REVIEW

File: resources/assets/js/components/countdown/script.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  name: 'countdown',
+  data() {
+    return {
+      startDate: 1527811201000,
+      days: null,
+      hours: null,
+      minutes: null,
+      message: null,
+      interval: null,
+      running: null,
```
Comment (loeffel-io) [inline resources/assets/js/components/countdown/script.js]: should be false

File: resources/assets/js/components/countdown/script.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+export default {
+  name: 'countdown',
+  data() {
+    return {
+      startDate: 1527811201000,
```
Comment (loeffel-io) [inline resources/assets/js/components/countdown/script.js]: maybe endDate is better

**Comment by loeffel-io** [comment]
Schrift zu klein

File: resources/assets/js/components/countdown/script.js (hunk 2 above)
Comment (kstar20001) [reply resources/assets/js/components/countdown/script.js]: depends on the point of view :). I thought about the startdate of the ico and not the enddate of the countdown

**Pull request merged**